agent/config/config_windows.go

```diff
@@ -57,6 +57,8 @@ func DefaultConfig() Config {
 	programData := utils.DefaultIfBlank(os.Getenv("ProgramData"), `C:\ProgramData`)
 	ecsRoot := filepath.Join(programData, "Amazon", "ECS")
 	dataDir := filepath.Join(ecsRoot, "data")
+	DefaultPauseContainerImageName = "amazonaws.com/ecs/pause-windows"
+	DefaultPauseContainerTag = "latest"
```

Comment:
why not move these default values into the const block?

Reply:
Yes, we have made the change. We have moved these values to const_windows.go
Additionally, changes have been made to check if any values are set in 'PAUSE_CONTAINER_IMAGE_NAME' and 'PAUSE_CONTAINER_IMAGE_TAG'. If so, then those values are taken instead of default values.

Reply:
As discussed with @sparrc , we have added the Windows pause image name and tag in the Makefile and corresponding changes have been made in build file to invoke Windows agent build using the loader flags.

Additionally, a Powershell script has been created which can build standalone agent on Windows.